syft/workers/worker_support.py

```diff
@@ -0,0 +1,22 @@
+from syft import dependency_check
+
+from syft.workers.base import BaseWorker
+
+from syft.frameworks.crypten.worker_support import add_support_to_worker, remove_support_from_worker
+
+
+supported_frameworks = {}
+
+if dependency_check.crypten_available:
+    supported_frameworks["crypten"] = {
+        "add_support": add_support_to_worker,
+        "remove_support": remove_support_from_worker,
+    }
```

Comment:
What about adding the framework name in these functions?

```suggestion
from syft.frameworks.crypten.worker_support import add_support_to_worker as add_crypten_support, remove_support_from_worker as remove_crypten_support


supported_frameworks = {}

if dependency_check.crypten_available:
    supported_frameworks["crypten"] = {
        "add_support": add_crypten_support,
        "remove_support": remove_crypten_support,
    }
```